### PR TITLE
2 party POL Astroport Single domain (Neutron) e2e test with manager

### PR DESCRIPTION
**Pull request by keyleu**
Adds the e2e test for the 2 party POL single domain using the manager.

Some services were using ActionsMsgs and others ActionMsgs, so changed them all to be the same (ActionMsgs).
Also added test to CI.

### CODE REVIEW

File: local-interchaintest/examples/two_party_pol_astroport_neutron.rs [hunk 1]
```diff
@@ -0,0 +1,1052 @@
...
+                    splits: vec![
+                        valence_reverse_splitter_service::msg::UncheckedSplitConfig {
+                            denom: UncheckedDenom::Native(token.clone()),
+                            account: ServiceAccountType::AccountId(1),
+                            amount:
+                                valence_reverse_splitter_service::msg::UncheckedSplitAmount::FixedAmount(
+                                    lp_amount.into(),
+                                ),
+                            factor: None,
+                        },
```
Comment (stiiifff) [inline local-interchaintest/examples/two_party_pol_astroport_neutron.rs:285]: You could use the `with_native_amount` helper function.

File: local-interchaintest/examples/two_party_pol_astroport_neutron.rs [hunk 2]
```diff
@@ -0,0 +1,1052 @@
...
+        test_ctx
+            .get_request_builder()
+            .get_request_builder(NEUTRON_CHAIN_NAME),
+        &neutron_receiver,
+    );
+
+    assert!(tokenfactory_token_balance.len() == 1);
+    assert!(neutron_balance.len() == 1);
+    assert!(tokenfactory_token_balance[0].denom == token);
+    assert!(neutron_balance[0].denom == *NTRN_DENOM);
```
Comment (stiiifff) [inline local-interchaintest/examples/two_party_pol_astroport_neutron.rs:1047]: Why not verify the balance amounts ?

**Comment by stiiifff** [comment]
It's really amazing to see all the Valence components coming together in this e2e test ! 🥳🚀

**Review by stiiifff — APPROVED** [review]

File: local-interchaintest/examples/two_party_pol_astroport_neutron.rs (hunk 1 above)
Comment (keyleu) [reply local-interchaintest/examples/two_party_pol_astroport_neutron.rs:285]: tried but can't because I'm using serviceId for the workflow not Addr

**Pull request merged**